ATAPHtmlReport/report.css

```diff
@@ -261,6 +261,38 @@ button{
 	font-family: Arial, sans-serif;
 }
 
+table {
```

Comment:
kann das nicht auch zu konflikten fuehren. also das gilt ja fuer alle table tags in der ganzen html. wenn fbpro jetzt iwo eine table hinzufuegt sieht die dann doch auch komisch aus oder?
besser sind hier ids oder klassen